includes/gateway/class-omise-payment-internetbanking.php

```diff
@@ -108,7 +108,7 @@ public function process_payment( $order_id ) {
 					'amount'      => $this->format_amount_subunit( $order->get_total(), $order->get_order_currency() ),
 					'currency'    => $order->get_order_currency(),
 					'description' => apply_filters('omise_charge_params_description', 'WooCommerce Order id ' . $order_id, $order),
-					'offsite'     => $_POST['omise-offsite'],
+					'source'      => array( 'type' => $_POST['omise-offsite'] ),
```

Comment:
Minor concern - is this the right way to get the from data in WooCommerce? Taking the stuff straight from `$_POST` makes me imagine potential security issues

Reply:
👍  Good point
There is one issue about sanitizing form submission, I'll make a fix about this form-post topic in that coming pull request in the very soon (in a plan before release v3.3).